Form-Controls/styles.css

```diff
@@ -0,0 +1,60 @@
+
+
+/* header */
+.head{
+    display: flex;
+    justify-content: center;
+}
+/* form */
+input:focus {

```

Comment:
I like this focus style! If you were to extend this to other states, how would you do this and what would those states be?